### PR TITLE
Resolve rate from coinGecko if exchange rate is not found

### PR DESCRIPTION
**Pull request by Xiaolei-Shawn**
## Description

- Resolve currency rate from `coinGecko` if its key is not found in `exchangeRates`.
- Fix a bug when calculating exchange rate that base and quote currency key are passed to function `newGetExchangeRatesForCurrencies` with wrong order: https://github.com/Kwenta/kwenta/blob/51068f9900e9f597adc85aabd9825ea014179a5c/hooks/useExchange.ts#L278
## Related issue
 
[Issue](https://github.com/Kwenta/kwenta/issues/1273)

## Motivation and Context

The title in exchange won't show correct rate if one or both currencies are not found in exchange rates results.

## How Has This Been Tested?

- Go to `Exchange` view
- Select `sINR` and `DAI`
- Tab title in browser should show correct exchange rate
- Switch currency to see title change

## Screenshots (if appropriate):
image
image

### CODE REVIEW

**Comment by platschi** [comment]
@Xiaolei-Shawn Kindly check lint errors :)

**Comment by Xiaolei-Shawn** [comment]
> @Xiaolei-Shawn Kindly check lint errors :)



> @Xiaolei-Shawn Kindly check lint errors :)

My bad! Fixed already.

**Comment by LeifuChen** [comment]
@Xiaolei-Shawn 
I think the function signature is ordered like this by some reason. The rate on title has been fixed but it caused the issue for the calculation of the received amount.
image

File: pages/exchange.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ const Exchange: ExchangeComponent = () => {
 						? t('exchange.page-title-currency-pair', {
 								baseCurrencyKey,
 								quoteCurrencyKey,
-								rate: formatCurrency(quoteCurrencyKey, inverseRate, {
+								rate: formatCurrency(quoteCurrencyKey, rate, {
```
Comment (LeifuChen) [inline pages/exchange.tsx:35]: As you may notice, it used the `inverseRate` on page title. It explains why the order of the function signature is a bit strange.

**Review by LeifuChen — CHANGES_REQUESTED** [review]

File: pages/exchange.tsx (hunk 1 above)
Comment (Xiaolei-Shawn) [reply pages/exchange.tsx]: Now revert to inverted rate

**Comment by LeifuChen** [comment]
@Xiaolei-Shawn The rate of synthetic assets (the assets with the prefix `s`, which is fetching the price from `rate` not via coincgecko api) is not working now.
image

**Review by LeifuChen — APPROVED** [review]
Looks good to me.

**Review by platschi — APPROVED** [review]
Perfect, works as expected.

**Pull request merged**